app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ProjectsController < ApplicationController
+
+  def show
+    @project = Project.find(params[:id])
+    @theme = @project.challenge_theme
+    @count = @project.con_count
+    @exp = @project.average_exp
```

Comment:
I see what you were wanting to do here but we don't want our controller to have more than it needs. So I would just have line 4 in here. You can reference all of the methods that you built in your view by calling `@project.challenge_theme`, etc.